src/games/strategy/triplea/delegate/BattleTracker.java

```diff
@@ -62,6 +65,7 @@
   // blitzed territories
   private final Set<Territory> m_blitzed = new HashSet<>();
   // territories where a battle occurred
+  // TODO: fix typo in name, 'fough' -> 'fought'
```

Comment:
Hate to say, but now that we are doing backwards compatible releases, we need to double check the blast radius of variable re-names when it comes to SaveGames. Anything that is a GameComponent must keep the same variable names for save games almost certainly. BattleTracker might be safe, not willing to check at the time. Also want to reduce code churn in case we choose to go ahead and fix the 'm_' naming as well.